installation/sql/postgresql/base.sql

```diff
@@ -119,7 +119,8 @@ INSERT INTO "#__assets" ("id", "parent_id", "lft", "rgt", "level", "name", "titl
 (93, 1, 171, 172, 1, 'com_fields', 'com_fields', '{}'),
 (94, 1, 173, 174, 1, 'com_workflow', 'com_workflow', '{}'),
 (95, 1, 175, 176, 1, 'com_guidedtours', 'com_guidedtours', '{}'),
-(96, 18, 130, 131, 2, 'com_modules.module.109', 'Guided Tours', '{}');
+(96, 18, 130, 131, 2, 'com_modules.module.109', 'Guided Tours', '{}'),
+(97, 1, 177, 178, 1, 'com_scheduler.task.1', 'com_scheduler.task.1', '{}');
 
 SELECT setval('#__assets_id_seq', 97, false);
```

Comment:
```suggestion
SELECT setval('#__assets_id_seq', 98, false);
```
This is why system tests fail in drone for PostgreSQL. When creating a category, an asset shall be created, and that would have id 97 which already exists.